### ISSUE TITLE
Error: Using this with Nuxt 3 fails

### ISSUE THREAD

**Issue by itszooper** [ssleptsov/ninja-keys#31]
I tried installing Ninja keys with Nuxt 3, but I keep getting lots of errors. Any ides on how to use this?

Here's a repro - https://stackblitz.com/edit/nuxt-starter-sgpcca?file=app.vue

**Comment by itszooper** [comment]
@ssleptsov Any idea on how to fix this?

**Comment by Mokkapps** [comment]
This is a working Nuxt 3 example if someone needs a working example: https://stackblitz.com/edit/github-q9qsjo?file=app.vue

@ssleptsov maybe we can link this as an official demo on the README?

**Comment by gsabater** [comment]
@itszooper and @Mokkapps 
If you plan to use it with nuxt, dont forget to add "ninja-keys" as a custom components in nuxt.config

```
vite: {
    vue: {
      template: {
        compilerOptions: {
          isCustomElement: (tag) => ['ninja-keys'].includes(tag),
        },
      },
    },
  },
```

**Comment by Mokkapps** [comment]
> @itszooper and @Mokkapps If you plan to use it with nuxt, dont forget to add "ninja-keys" as a custom components in nuxt.config
> 
> ```
> vite: {
>     vue: {
>       template: {
>         compilerOptions: {
>           isCustomElement: (tag) => ['ninja-keys'].includes(tag),
>         },
>       },
>     },
>   },
> ```

Yes, good point. I updated my StackBlitz example with this information.
